FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -141,41 +156,63 @@ public override void OnFixedUpdate()
             }
         }
 
```

Comment:
Personally I'm not all that sure about this function, but it should work most of the time ...